target/linux/mvebu/Makefile

```diff
@@ -11,7 +11,7 @@ BOARD:=mvebu
 BOARDNAME:=Marvell Armada 37x/38x/XP
 FEATURES:=usb pci pcie gpio nand squashfs
 CPU_TYPE:=cortex-a9
-CPU_SUBTYPE:=vfpv3
+CPU_SUBTYPE:=neon
```

Comment:
As far as I can tell this will break any Armada XP devices like the WRT1900ACv1, as they don't support Neon. E.g. http://www.marvell.com/embedded-processors/assets/Marvell_ARMADA_SOC_Prestera_Solutions_Brochure_4web.pdf  describes the Armada 38x as "Single or dual core ARM v7-Coretex-A9 CPU with NEON SIMD and Point Unit (FPU)", while Armada XP is described as "ARM v7-compliant Core Processor (CPU) that includes Marvell micro-architecture enhancements and a double-precision IEEE-compliant Floating Point Unit (FPU) per core" without any mention of NEON support.